src/Symfony/Component/DomCrawler/Tests/FormTest.php

```diff
@@ -847,7 +853,7 @@ protected function getFormFieldMock($name, $value = null)
     protected function createForm($form, $method = null, $currentUri = null)
     {
         $dom = new \DOMDocument();
-        $dom->loadHTML('<html>'.$form.'</html>');
```

Comment:
Why did you make this change?

Reply:
Now I'm testing form with HTML5 element. Without `@` `loadHTML` method will return warning type of `Tag template invalid in Entity...`